waiter/config-full.edn

```diff
@@ -482,6 +482,9 @@
                                                            :default-container-image "twosigma/waiter-test-apps:latest"
                                                            ;; The default namespace to use for Kubernetes objects created by Waiter
                                                            :default-namespace "waiter"
+                                                           ;; Function that determines if the fileserver container should be enabled for a service.
+                                                           ;; The function takes the service description as input.
+                                                           :fileserver-container-enabled? waiter.some.package/fileserver-enabled?
```

Comment:
I strongly prefer that this be implemented as a `:predicate-fn` inside the `:fileserver` config like we did with the `:reverse-proxy` for consistency.

Reply:
We would also want this function to have the same signature as the `:reverse-proxy :predicate-fn` for consistency and flexibility.

Reply:
sgtm, will change